frontends/p4/toP4/toP4.cpp

```diff
@@ -241,6 +241,9 @@ bool ToP4::preorder(const IR::Type_Void *) {
 }
 
 bool ToP4::preorder(const IR::Type_Name *t) {
+    auto cmts = t->srcInfo.getComments();
+    builder.append(cmts);
```

Comment:
You are seeing these failures because this always prints a newline unconditionally. Would suggest only adding it when there are comments.